docs/control-structures.rst

```diff
@@ -488,3 +492,91 @@ In the above example, ``revert("Not enough Ether provided.");`` returns the foll
 .. note::
     There used to be a keyword called ``throw`` with the same semantics as ``revert()`` which
     was deprecated in version 0.4.13 and removed in version 0.5.0.
+
+``try``/``catch``
+-----------------
+
+A failure in an external call can be caught using a try/catch statement, as follows:
+
+::
+
+    pragma solidity >=0.5.0 <0.7.0;
```

Comment:
`>=0.6.0`?

Reply:
That does not work because we are still at 0.5.12 in the develop_060 branch.